source/gameengine/Ketsji/KX_ConstraintWrapper.h

```diff
@@ -33,14 +33,14 @@
 #define __KX_CONSTRAINTWRAPPER_H__
 
 #include "EXP_Value.h"
+#include "PHY_DynamicTypes.h"
 
 class PHY_IConstraint;
 
 class	KX_ConstraintWrapper : public CValue
 {
 	Py_Header
-public:
-	KX_ConstraintWrapper(PHY_IConstraint *constraint);
+public: KX_ConstraintWrapper(PHY_IConstraint *constraint, PHY_ConstraintType ctype, int user_id);
```

Comment:
I think it is better to put the KX_ConstraintWrapper in the next line after public: not in the same line.